doc/index.rst

```diff
@@ -110,7 +110,9 @@ Documentation
 
 * :doc:`introduction`
 * :doc:`installation`
+* :doc:`short_tutorial`
 * :doc:`tutorial`
+* :doc:`data_workflow`
 * :doc:`notebooks`
```

Comment:
It should be a good idea to change the name of the notebooks section, or maybe integrate the notebooks section into the file tutorial.rst

Maybe nice to revise the name as "notebooks" is a technical word, they may be explanatory examples but I'm fine with the current name for the time being

Reply:
Changed to "Examples"